src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Loads task data from storage
+     *
+     * @return ArrayList of tasks
+     * @throws DukeException
```

Comment:
Feel like the condition when the `DukeException` is thrown should be specified here :))